Assets/Content/Systems/Player/IK/SimpleBodyPartLookAt.cs

```diff
@@ -1,89 +1,93 @@
 ﻿using System;
 using UnityEditor;
 using UnityEngine;
-public class SimpleBodyPartLookAt : MonoBehaviour
+
+namespace Content.Systems.Player.IK
 {
-    [SerializeField] public Camera camera;
-    [SerializeField] public Transform target;
-    private Vector3 mousePos = Vector3.zero;
+    /// <summary>
+    /// This class is used to find where the mouse position is and move a target to it.
+    /// Will be removed eventually
+    /// </summary>
+    public class SimpleBodyPartLookAt : MonoBehaviour
+    {
+        [SerializeField] public Camera camera;
+        [SerializeField] public Transform target;
+        private Vector3 mousePos = Vector3.zero;
 
-    [SerializeField] public Quaternion currentRot;
+        [SerializeField] public Quaternion currentRot;
 
-    [SerializeField] public float minRotationLimit;
-    [SerializeField] public float maxRotationLimit;
+        [SerializeField] public float minRotationLimit;
+        [SerializeField] public float maxRotationLimit;
 
-    [Range(0.5f, 10)]
-    [SerializeField] public float rotationSpeed = 5;
+        [Range(0.5f, 10)]
+        [SerializeField] public float rotationSpeed = 5;
 
-    [SerializeField] public Limits limits;
+        [SerializeField] public Limits limits;
 
-    [Serializable]
-    public class Limits
-    {
-        public bool x;
-        public bool y;
-        public bool z;
-    }
+        [Serializable]
+        public class Limits
+        {
+            public bool x;
+            public bool y;
+            public bool z;
+        }
 
 #if UNITY_EDITOR
-    // This class is resposable of displaying the bools properly. Values are hardcoded for lack of better means. (Currently unused)
-    //[CustomPropertyDrawer(typeof(Limits))]
-    private class LimitsUIE : PropertyDrawer
-    {
-        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        // This class is resposable of displaying the bools properly. Values are hardcoded for lack of better means. (Currently unused)
```

Comment:
What does "Currently unused" mean?
```suggestion
        // This class is responsible for displaying the bools properly. Values are hardcoded for lack of better means. (Currently unused)
```